packages/@headlessui-react/src/components/combobox/combobox.tsx

```diff
@@ -816,6 +831,10 @@ function ComboboxFn<TValue, TTag extends ElementType = typeof DEFAULT_COMBOBOX_T
     onClose?.()
   })
 
+  let setIsTyping = useEvent((isTyping = true) => {
```

Comment:
No need for the default value. We always pass a value in.

Reply:
True, but this was shorter than `: boolean` 🤭

I'll change it haha

Reply:
lol fair